include/deal.II/base/thread_local_storage.h

```diff
@@ -158,29 +147,21 @@ namespace Threads
     void
     clear();
 
+  private:
     /**
-     * Return a reference to the internal Threading Building Blocks
-     * implementation. This function is really only useful if deal.II has been
-     * configured with multithreading and has no useful purpose otherwise.
+     * The data element we store.
      */
-#  ifdef DEAL_II_WITH_THREADS
-    tbb::enumerable_thread_specific<T> &
-#  else
-    T &
-#  endif
-    get_implementation();
```

Comment:
This is strictly speaking an incompatible change. What about we deprecate the `get_implementation` method quickly in the 9.2 release?

Reply:
I think we are done with 9.2.

Reply:
Let's not do anything with 9.2 any more. But I wouldn't worry about it -- it's surely a class where we can afford to remove an obscure function without a deprecation period.